migrations/sql/R__stored_procedure_for_mine_information.sql

```diff
@@ -352,11 +366,20 @@ CREATE OR REPLACE FUNCTION transfer_mine_information() RETURNS void AS $$
             INSERT INTO ETL_LOCATION(
                 mine_guid           ,
                 mine_no             ,
+                mine_location_description,
                 latitude            ,
                 longitude           )
             SELECT
                 new.mine_guid,
                 new.mine_no  ,
+                COALESCE(
```

Comment:
What does 'Coalesce' do?

Reply:
Basically a wrapper that says, execute this SQL and if it returns NULL then do something else, if you dont do anything else, imma just return NULL.
Since in my case I want it to return NULL if the query doesnt resolve, I left it at that.
http://www.postgresqltutorial.com/postgresql-coalesce/

Reply:
thanks for the link